pkg/blobfuse-proxy/blobfuse-proxy.service

```diff
@@ -2,7 +2,7 @@
 Description=Blobfuse proxy service
 
 [Service]
-ExecStart=/usr/bin/blobfuse-proxy --v=5 --blobfuse-proxy-endpoint=unix://var/lib/kubelet/plugins/blob.csi.azure.com/blobfuse-proxy.sock
+ExecStart=/usr/local/bin/blobfuse-proxy --v=5 --blobfuse-proxy-endpoint=unix://var/lib/kubelet/plugins/blob.csi.azure.com/blobfuse-proxy.sock
```

Comment:
since this path has been changed, what's the impact if user upgrades from old version to this version?